### ISSUE TITLE
PyPI package for modularize-2 branch

### ISSUE THREAD

**Issue by gdunstone** [sivel/speedtest-cli#314]
Is it possible to get a PyPI package for the modularize-2 branch or is a merge imminent for the whole project (to make it generally modular)?
I'm using the modularize-2 branch in another project and it would be tops if either there could be a PyPI package or if a merge into master was close enough that we don't really have to worry about it.

https://github.com/enkore/i3pystatus/pull/449#issuecomment-250068455

**Comment by sivel** [comment]
This is coming. The branch is not complete yet and is still a work in progress. Once it is finished it will get merged to devel and eventually into master and released.
